src/lib/modules/debugger/index.ts

```diff
@@ -332,14 +336,18 @@ class TransactionDebugger {
           this.embark.logger.warn(NO_DEBUG_SESSION);
           return callback();
         }
-        this.cmdDebugger.displayGlobals();
+        this.embark.logger.info("Globals:");
+        const debugVars = this.simplifyDebuggerVars(this.cmdDebugger.solidityState);
+        for (const debugVar of Object.keys(debugVars)) {
+          this.embark.logger.info(`${debugVar}: ` + `${debugVars[debugVar]}`.white);
+        }
```

Comment:
We could put that in a function because it's used for global and local and below too